openghg/standardise/surface/_icos.py

```diff
@@ -125,7 +129,7 @@ def _read_data_large_header(
     len_header = len(header)
 
     if len_header != 40:
-        print(
+        logger.warning(
```

Comment:
Similiarly to above, can probably removed the word "WARNING" from the string?

Reply:
Done.